src/editor.js

```diff
@@ -1665,21 +1670,28 @@ export class OdooEditor extends EventTarget {
         const cursor = this._historySteps[this._historySteps.length - 1].cursor;
         const { focusOffset, focusNode, anchorNode, anchorOffset } = cursor || {};
         const wasCollapsed = !cursor || (focusNode === anchorNode && focusOffset === anchorOffset);
+
+        // Sometimes google chrome wrongly triggers an input event with `data`
+        // being `null` on `deleteContentForward` `insertParagraph`. Luckily,
+        // chrome provide the proper signal with the event `beforeinput`.
+        const chromeDeleteforward =
```

Comment:
This is absolute total nitpicking but I prefer when constants that hold a boolean are named like closed questions: `isChromeDeleteForward`.

Reply:
I agree, I usually do the same, I forgot this time. :+1: